ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -192,6 +192,11 @@ object Cli {
       .text("Enables JWT-based authorization, where the JWT is signed by RSA256 with a public key loaded from the given JWKS URL")
       .action( (url, config) => config.copy(authService = Some(AuthServiceJWT(JwksVerifier(url)))))
 
+    opt[Unit]("sortable-ci")
```

Comment:
rename to `sortable-contractids`.

Reply:
I'd also say we should make it `hidden()` and only ask early voluntary users to turn it on. No need to surface it in the CLI.

Reply:
Fixed. Fixed